packages/studio/src/utils/codemirror.utils.ts

```diff
@@ -31,13 +29,15 @@ export const myTheme = EditorView.theme({
     }
 }, {dark: true})
 
-export const language = new Compartment 
+export const language = new Compartment
 
 export const myHighlightStyle = HighlightStyle.define([
-  {tag: customTags.chord, color: "#0481aa"},
+  {tag: tags.keyword, color: "#0481aa"},
```

Comment:
I replaced the custom tag with the closest thing I could find in the existing highlighter tags just to keep things simple. This will also allow using existing codemirror themes (not that we want to, but leaves the door open).

Reply:
Good with me for now. 

I think all of the colors in this file for theme could ultimatley end up in a config. Whether that's the state, or in some optional prop you could pass to the component. That way we could allow the configuration of the theme, without being restricted by names that don't line up with what the thing actually is. Then we'd have docs on how to style the editor etc. 

I could be thinking that there's going to be alot more highlight tags then there actually will be as well.